receiver/awscontainerinsightreceiver/internal/stores/servicestore.go

```diff
@@ -82,3 +82,7 @@ func addServiceNameTag(metric CIMetric, serviceNames []string) {
 	// TODO handle serviceNames len is larger than 1. We need to duplicate the metric object
 	metric.AddTag(ci.TypeService, serviceNames[0])
 }
+
+func (s *ServiceStore) Shutdown() error {
```

Comment:
Why is this needed?

Reply:
I was trying to make it consistent with other store even though this will be a no-op. Probably better to remove.